telegram/ext/_application.py

```diff
@@ -337,6 +337,22 @@ def concurrent_updates(self) -> int:
         """
         return self._concurrent_updates
 
+    @property
+    def job_queue(self) -> Optional["JobQueue"]:
+        """
+        :class:`telegram.ext.JobQueue`: The :class:`JobQueue` used by the
```

Comment:
The lines on `job_queue` in the `Attrs` block of the class-docstring can be removed, then :)